yt/geometry/grid_geometry_handler.py

```diff
@@ -208,19 +208,17 @@ def print_stats(self):
             if (self.level_stats["numgrids"][level]) == 0:
                 continue
             print(
-                "% 3i\t% 6i\t% 14i\t% 14i"
-                % (
-                    level,
-                    self.level_stats["numgrids"][level],
-                    self.level_stats["numcells"][level],
-                    np.ceil(self.level_stats["numcells"][level] ** (1.0 / 3)),
-                )
+                f"{level:>3}\t"
+                f"{self.level_stats['numgrids'][level]:>6}\t"
+                f"{self.level_stats['numcells'][level]:>14}\t"
+                f"{np.ceil(self.level_stats['numcells'][level] ** (1.0 / 3)):>14}"
```

Comment:
`np.ceil` will return a float here -- would need  to cast to int here to keep it identical 

```suggestion
                f"{int(np.ceil(self.level_stats['numcells'][level] ** (1.0 / 3))):>14}"
```